packages/pymedphys/src/pymedphys/dicom.py

```diff
@@ -13,4 +13,4 @@
     dicom_dataset_from_dict
 )
 
-from pymedphys_analysis.geometry import get_structure_aligned_cube
+# from pymedphys_analysis.geometry import get_structure_aligned_cube
```

Comment:
@Centrus007 the docs use this don't they?

Reply:
We just need to remove `get_structure_aligned_cube` from the docs for now I think.

Reply:
The following really needs to be fixed:

https://github.com/pymedphys/pymedphys/issues/340 ...